README.md

```diff
@@ -14,10 +18,57 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### Node.js
+
+If you want to install Node.js natively visit [Node.js](http://nodejs.org), please make sure it is version 4.4.4
+
+To check the locally installed version run: 
+
+```bash
+node --version
 ```
-git clone https://github.com/department-of-veterans-affairs/healthcare-application.git
-cd healthcare-application
+
+Once you have version 4.4.4 of Node.js installed to required version of npm run:
```

Comment:
This sentence doesn't make sense. I think what you're trying to say is: `Once you have version 4.4.4 of Node.js installed, require version 3.8.9 of npm by running:`